internal/perms/perms.go

```diff
@@ -17,35 +17,21 @@ package perms
 
 import (
 	"fmt"
-	"os/user"
-	"strconv"
+	"os"
 )
 
 // MyUserAndGroup returns the UID and GID of this process.
 func MyUserAndGroup() (uid uint32, gid uint32, err error) {
-	// Ask for the current user.
-	user, err := user.Current()
```

Comment:
An improvement idea is to depend on the partial output returned by `user.Current()` even when if returns error . This is based on internal comment https://yaqs.corp.google.com/eng/q/7089520134015942656#a1n6 .

Reply:
Captured it in the code.